schedule-replicas-by-topology-labels.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 通过拓扑 label 进行副本调度
+aliases: ['/docs-cn/v3.1/location-awareness/','/docs-cn/v3.1/how-to/deploy/geographic-redundancy/location-awareness/','/zh/tidb/v3.1/location-awareness']
+---
+
+# 通过拓扑 label 进行副本调度
+
+为了提升 TiDB 集群的高可用性和数据容灾能力，我们推荐让 TiKV 节点尽可能在物理层面上分散，例如让 TiKV 节点分布在不同的机架甚至不同的机房。PD 调度器根据 TiKV 的拓扑信息，会自动在后台通过调度使得 Region 的各个副本尽可能隔离，从而使得数据容灾能力最大化。
+
+要让这个机制生效，需要在部署时进行合理配置，把集群的拓扑信息（特别是 TiKV 的位置）上报给 PD。阅读本章前，请先确保阅读 [TiDB Ansible 部署方案](/online-deployment-using-ansible.md)。
```

Comment:
`Docker deployment` is missing. Is it expected?

Reply:
We no longer maintain the `Docker composer` deployment method, so I prefer removing it here. 
![image](https://user-images.githubusercontent.com/41534398/93437464-9990b280-f8fe-11ea-8fc1-f24a757662d5.png)

Reply:
OK.